### PR TITLE
WIP Ingress-gce helm chart

### PR DESCRIPTION
**Pull request by kron4eg**
**How to categorize this PR?**

/area control-plane
/area networking
/kind enhancement
/platform gcp

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator

```

### CODE REVIEW

**Comment by gardener-robot-ci-2** [comment]
Thank you @kron4eg for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

File: charts/internal/shoot-system-components/charts/default-http-backend/templates/rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  verbs: ["update", "patch"]
+- apiGroups: ["networking.istio.io"]
+  resources: ["destinationrules"]
+  verbs: ["get", "list", "watch", "update", "patch"]
+- apiGroups: ["extensions", "networking.k8s.io"]
+  resources: ["ingresses"]
+  verbs: ["get", "list", "watch"]
+# For now, GLBC annotates ingress resources with various state and statuses:
+# https://github.com/kubernetes/ingress-gce/blob/50d49b077d9ab4362a02fae05f94e433cd3f08dc/pkg/controller/controller.go#L579
+# TODO(rramkumar1): Remove unnecessary `update` permission once statuses are propagated through `ingresses/status`
```
Comment (AndreasBurger) [inline charts/internal/shoot-system-components/charts/default-http-backend/templates/rbac.yaml:61]: I don't think we need to include the upstream TODOs. Same in l.70

File: charts/internal/seed-controlplane/values.yaml [hunk 2]
```diff
@@ -5,3 +5,5 @@ cloud-controller-manager:
   enabled: true
 csi-driver-controller:
   enabled: true
+ingress-gce:
+  enabled: true
```
Comment (AndreasBurger) [inline charts/internal/seed-controlplane/values.yaml:9]: From what I saw during testing/playing around with this controller, I'd rather not have it enabled by default. In my opinion that should be an option, unless there are cases where it _needs_ to be deployed.

**Comment by kron4eg** [comment]
Superseded by #848

**Pull request closed without merging**